pkg/sdk/connections_gen.go

```diff
@@ -0,0 +1,121 @@
+	create      bool                     `ddl:"static" sql:"CREATE"`
+	connection  bool                     `ddl:"static" sql:"CONNECTION"`
+	IfNotExists *bool                    `ddl:"keyword" sql:"IF NOT EXISTS"`
+	name        AccountObjectIdentifier  `ddl:"identifier"`
+	ReplicaOf   ExternalObjectIdentifier `ddl:"identifier" sql:"AS REPLICA OF"`
+	Comment     *string                  `ddl:"parameter,single_quotes" sql:"COMMENT"`
+}
+
+// AlterFailoverConnectionOptions is based on https://docs.snowflake.com/en/sql-reference/sql/alter-connection.
+type AlterFailoverConnectionOptions struct {
+	alter                     bool                       `ddl:"static" sql:"ALTER"`
+	connection                bool                       `ddl:"static" sql:"CONNECTION"`
+	name                      AccountObjectIdentifier    `ddl:"identifier"`
+	EnableConnectionFailover  *EnableConnectionFailover  `ddl:"keyword" sql:"ENABLE FAILOVER TO ACCOUNTS"`
+	DisableConnectionFailover *DisableConnectionFailover `ddl:"keyword" sql:"DISABLE FAILOVER"`
+	Primary                   *bool                      `ddl:"keyword" sql:"PRIMARY"`
+}
+type EnableConnectionFailover struct {
+	ToAccounts         []AccountIdentifier `ddl:"list,no_parentheses"`
+	IgnoreEditionCheck *bool               `ddl:"keyword" sql:"IGNORE EDITION CHECK"`
+}
+type DisableConnectionFailover struct {
+	ToAccounts *ToAccounts `ddl:"keyword" sql:"TO ACCOUNTS"`
+}
+type ToAccounts struct {
+	Accounts []AccountIdentifier `ddl:"list,no_parentheses"`
+}
+
+// AlterConnectionOptions is based on https://docs.snowflake.com/en/sql-reference/sql/alter-connection.
+type AlterConnectionOptions struct {
+	alter      bool                    `ddl:"static" sql:"ALTER"`
+	connection bool                    `ddl:"static" sql:"CONNECTION"`
+	IfExists   *bool                   `ddl:"keyword" sql:"IF EXISTS"`
+	name       AccountObjectIdentifier `ddl:"identifier"`
+	Set        *Set                    `ddl:"keyword" sql:"SET"`
+	Unset      *Unset                  `ddl:"keyword" sql:"UNSET"`
+}
+type Set struct {
+	Comment *string `ddl:"parameter,single_quotes" sql:"COMMENT"`
+}
+type Unset struct {
+	Comment *bool `ddl:"keyword" sql:"COMMENT"`
+}
+
+// DropConnectionOptions is based on https://docs.snowflake.com/en/sql-reference/sql/drop-connection.
+type DropConnectionOptions struct {
+	drop       bool                    `ddl:"static" sql:"DROP"`
+	connection bool                    `ddl:"static" sql:"CONNECTION"`
+	IfExists   *bool                   `ddl:"keyword" sql:"IF EXISTS"`
+	name       AccountObjectIdentifier `ddl:"identifier"`
+}
+
+// ShowConnectionOptions is based on https://docs.snowflake.com/en/sql-reference/sql/show-connections.
+type ShowConnectionOptions struct {
+	show        bool  `ddl:"static" sql:"SHOW"`
+	connections bool  `ddl:"static" sql:"CONNECTIONS"`
+	Like        *Like `ddl:"keyword" sql:"LIKE"`
+}
+type connectionRow struct {
+	SnowflakeRegion           string         `db:"snowflake_region"`
```

Comment:
Not all of the fields are mapped according to https://docs.snowflake.com/en/sql-reference/sql/show-connections#output where some of the fields like `region_group` can appear for organizations that span multiple region groups.